scripts/gdprdeleteuser.php

```diff
@@ -0,0 +1,114 @@
+#!/usr/local/bin/php -q
+<?php
+/**
+ * DO NOT RUN THIS SCRIPT WILLY NILLY
+ * This script will completely yoink a users personal details, show credits, podcast credits and a bunch of other stuff.
+ * Only run this script if a user has explicity declared they wish to be forgotten and understand the consequences.
+ * 
+ * run with "php gdprdeleteuser.php [userid]"
+ */
+
+use \MyRadio\Database;
+use \MyRadio\MyRadioException;
+use \MyRadio\MyRadioError;
+use \MyRadio\MyRadioEmail;
+use \MyRadio\MyRadio\CoreUtils;
+use \MyRadio\ServiceAPI\MyRadio_User;
+
+ini_set('log_errors', 1);
+ini_set('error_log', '/tmp/php-mailparser-error.log');
+ini_set('display_errors', 'On');
+
+require_once '../src/Controllers/root_cli.php';
+
+$db = Database::getInstance();
+
+$time = strtotime("-1 year", time());
+$date = date("Y-m-d", $time);
+
+$userid = $argv[1];
+
+try{
+    $db->query(
+        'INSERT INTO public.member(
+            memberid, fname, sname, college, receive_email, data_removal)
+            VALUES (1350883, \'deleted\', \'user\', 10, false, \'deleted\')',
+        []
+    );
+}  catch (exception $e) {
+    echo 'deleting user\n';
+}
+$textMemberList = implode(',', $memebersToDelete);
+
+$db->query(
+    'UPDATE schedule.show_credit SET memberid=1350883 WHERE memberid=$1',
```

Comment:
what is this 1350883 ? Probably should be a named constant?

Reply:
It has to be an int for database reasons. 1350883 is just a futureproofed number made by smashing together our fm and am frequencies.

Reply:
change made